programs/test/benchmark.c

```diff
@@ -589,19 +651,33 @@ int main( int argc, char *argv[] )
         mbedtls_ctr_drbg_context ctr_drbg;
 
         mbedtls_ctr_drbg_init( &ctr_drbg );
+        mbedtls_snprintf( title, sizeof( title ), "CTR_DRBG (NOPR)" );
```

Comment:
Why change to `mbedtls_snprintf` instead of passing the string directly to `TIME_AND_TSC`?

Reply:
so in case of error, we will print the title before we call `PRINT_ERROR`.
This will change, after I rebase, and the code from https://github.com/ARMmbed/mbedtls/commit/51d7cfe026fa3f33d86a80d27ea9013830f33992 will be introduced